src/main/java/task/DoAfter.java

```diff
@@ -0,0 +1,23 @@
+package task;
+
+public class DoAfter extends Tasks {
+    String after;
+
+    public DoAfter(String description, String type, String after) {
+        super(description, type);
+        this.after = after;
+    }
+
+    public String getAfter() {
+        return after;
+    }
+
+    public void setAfter(String after) {
+        this.after = after;
+    }
+
+    public String toMessage() {
+        return description
+            + "(after: " + after + ")";
+    }
+}
```

Comment:
Since you are doing a movies/tv shows related app, are these Duke tasks relevant still? 

If these codes are not used, remove them as these codes have no added values by being there, and adds confusions as to the purpose of these codes being here.

Same for those below (other duke stuffs).